backend/molgenis-emx2-jobs/src/main/java/org/molgenis/emx2/tasks/Step.java

```diff
@@ -24,13 +24,15 @@
   private Integer total;
   private long start = System.currentTimeMillis();
   long end;
+  private boolean strict = false;
```

Comment:
Why do we need this in the Step class?

Reply:
because I somehow need to make it trickle down into substeps

Reply:
I see you call isStrict() only at a task level. Can't the boolean reside in the Task class? So one level up? In the steps you do not seem to use it?

Reply:
Yup, it seems it can. Will move Step.strict to Task.strict